Update boardMapper.xml

aurevoir-jaeho opened the pull request:
<!-- 6-2) 판매글 삭제 -->
<!-- 6-3) 판매글 리플달기 -->
<!-- 6-4) 판매글 구매하기 -->
xml 코드 작성 완료. (검사 필요)

aurevoir-jaeho committed:
Update boardMapper.xml

<!-- 6-2) 판매글 삭제 -->
<!-- 6-3) 판매글 리플달기 -->
<!-- 6-4) 판매글 구매하기 -->
xml 코드 작성 완료. (검사 필요)

Lee-KS-kr reviewed (changes requested):
6-3번 insert문과 value의 갯수가 달라요 수정부탁드립니다

aurevoir-jaeho committed:
Update boardMapper.xml

<!-- 6-3) 판매글 리플달기 -->
수정 ^^

aurevoir-jaeho committed:
Merge branch 'main' into KJH

aurevoir-jaeho committed:
Update boardMapper.xml

<!-- 6-3) 판매글 리플달기 -->
<!-- 6-4) 판매글 구매하기 -->
오타 수정 ^^

Lee-KS-kr reviewed (changes requested):
44번 라인(select 첫번째 요소) 앞에 컴마가 있어요 지워주세용..!
77번라인 replynum은 sequence이용할거에요!
또다른 수정사항 추가 ---
7번 검색하기
parameter는 map으로 보낼거에요(key : category, item)
map이 null이 아닐 시 category는 category의 value이고
제목 혹은 내용에 item의 value를 like로 일치하게 검색하는 쿼리로 작성해주세요

aurevoir-jaeho committed:
Update boardMapper.xml

수정 요구사항 수행
--> 44번 라인(select 첫번째 요소) 앞에 컴마가 있어요 지워주세용..!
77번라인 replynum은 sequence이용할거에요!
또다른 수정사항 추가 ---
7번 검색하기
parameter는 map으로 보낼거에요(key : category, item)
map이 null이 아닐 시 category는 category의 value이고
제목 혹은 내용에 item의 value를 like로 일치하게 검색하는 쿼리로 작성해주세요
